src/components/atoms/Course.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface courseIdentification {
+    courseNumber: string;
+    courseName: string;
+}
+
+const Course = ({courseNumber, courseName}:courseIdentification) => {
+  return (
+    <div className="flex-1 text-left">
+        <p className="font-semibold">{courseNumber}</p>
+        <p className='text-xs mt-1'>{courseName}</p>
```

Comment:
```suggestion
        <p className='text-xs'>{courseName}</p>
```

instead of mt-1 put `spacing-y-1` in the parent div